src/vs-evm.md

```diff
@@ -0,0 +1,53 @@
+# FuelVM vs. EVM, Explained
+
+FuelVM learns from the EVM, Solana, WASM, Bitcoin, and Cosmos.
+
+This page is meant to outline the ways the FuelVM differs compared to the EVM in simple terms.
+
+## FuelVM vs. EVM
+
+### The FuelVM has a globally shared memory architecture instead of context-local memory
+
+The FuelVM has a globally shared memory architecture. Instead of every contract call having its own separate memory space, call data, and return data, all contract call frames share global memory. This chunk of memory is shared amongst all call frames and is globally readable. This allows you to pass data around between contracts without expensive storage and pass chunks of data without having to serialize, copy from call data to memory, etc. Read more about the FuelVM memory model [here](./fuelvm/memory_model.md).
+
+### The FuelVM is designed for fraud-provability
+
+The EVM is a complicated machine to construct fraud proofs for. It usually requires a second layer such as WASM or MIPS to be interpreted into a fraud provable system. Check out [User Sovereignty with Fraud Proofs](./why-fuel.md) and [how fraud proofs unlock key functionality](http://localhost:3000/modular-movement.html?highlight=fraud%20proo#trust-minimized-light-clients).
+
+### FuelVM has multiple native assets
+
+In Ethereum, the only native asset is Ether. It’s the only one that gets first-class treatment in terms of cost and ability to be pushed and pulled through a call. In Fuel, any contract can mint its UTXO-based native asset using a set of easy asset opcodes. All of which can gain the benefits of native-level call and optimization. Read more about support for multiple native assets in [the Sway docs](https://fuellabs.github.io/sway/v0.16.2/blockchain-development/native_assets.html), and [here](./fuelvm/native_assets.md)
+
+### FuelVM uses 64-bit words instead of 256-bit
+
+Modern processors have 64bit registers, and all of the instruction set operates on 64bits. Those are the most efficient instructions, and when you deal with 256 bits, you’re dealing with big numbers, and since modern processors aren't made to handle those numbers natively, it means you have to do more in the software.
```

Comment:
```suggestion
Modern processors have 64-bit registers, and all of the instruction set operates on 64 bits. Those are the most efficient instructions, and when you deal with 256 bits, you’re dealing with big numbers, and since modern processors aren't made to handle those numbers natively, it means you have to do more in the software.
```